source/renderer/app/stores/StakingStore.js

```diff
@@ -21,6 +21,8 @@ import StakePool from '../domains/StakePool';
 import REWARDS from '../config/stakingRewards.dummy.json';
 
 export default class StakingStore extends Store {
+  STAKE_POOLS_INITIAL_INTERVAL = 1000; // 1 second | unit: 1000 milliseconds
```

Comment:
Any reason why we have these variables as member variables of `StakingStore` class?
If these are constants then no need to store them as member of class.
We can declare them in constants file, and export for external usage.

Reply:
@yakovkaravelov can you please fix that?